tsl/test/expected/decompress_vector_qual.out

```diff
@@ -177,14 +179,67 @@ select * from vectorqual where ts > '2021-01-01 00:00:00' or metric3 > 40 order
  Wed Jan 01 00:00:00 2025 |      52 |      5 |      53 |      54 | tag5
 (5 rows)
 
+-- Some more tests for boolean operations.
+select count(*) from vectorqual where ts > '2021-01-01 00:00:00';
+ count 
+-------
+     3
+(1 row)
+
+select count(*) from vectorqual where 40 < metric3;
+ count 
+-------
+     4
+(1 row)
+
+select count(*) from vectorqual where metric2 < 0;
```

Comment:
Would be something like `metric2 IS NOT NULL` and `metric2 > NULL` also handled by vectorized logic?

Reply:
The first predicate seems to be tested in #6460, but might be good to have something for the second predicate as well.

Reply:
`metric2 > NULL` will be evaluated to constant false at planning time, because all comparison operators we vectorize are strict (a null in inputs leads to null output).